misk/src/main/kotlin/misk/tracing/backends/jaeger/JaegarSamplerConfig.kt

```diff
@@ -0,0 +1,9 @@
+package misk.tracing.backends.jaeger
+
+import misk.config.Config
+
+data class JaegerSamplerConfig (
+  val type: String?,
```

Comment:
what do null values mean for these configs? we should document them

Reply:
or give them defaults?